Tests/Particles/SOAParticle/main.cpp

```diff
@@ -0,0 +1,190 @@
+#include <AMReX.H>
+#include <AMReX_Particle.H>
+#include <AMReX_ParticleContainer.H>
+#include <AMReX_ParticleTile.H>
+#include <AMReX_ParIter.H>
+#include <AMReX_REAL.H>
+#include <AMReX_Vector.H>
+#include <AMReX_GpuContainers.H>
+
+#include <array>
+
+using namespace amrex;
+
+template <typename T_PC,template<class> class Allocator=DefaultAllocator>
+void addParticles ()
+{
+    int is_per[AMREX_SPACEDIM];
+    for (int d = 0; d < AMREX_SPACEDIM; d++)
+        is_per[d] = 1;
```

Comment:
linter:
```suggestion
    for (int & d : is_per)
        d = 1;
```

Reply:
Or everything in one line `int is_per[] = {AMREX_D_DECL(1,1,1)};`